models/engine/db_storage.py

```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class and its ID"""
+        objects = self.all(cls).values()
+        object_id = [item for item in objects if item.id == id][0]
+        if object_id:
+            return object_id
+        else:
```

Comment:
You can remove this else in order to the code be shorter

Reply:
I removed the else condition and put the return instead